generator.rb

```diff
@@ -0,0 +1,68 @@
+# Make "is_vowel?" method
+
+def is_vowel?(test_vowel)
+  case test_vowel.downcase
+  when "a", "e", "i", "o", "u"
+    return true
+  else
+    return false
+  end
+end
+
+# Make "run_generator" method
+def run_generator
+  puts "What is the first word to combine?"
+  word_1 = gets.chomp
+
+  until word_1.length > 1
```

Comment:
Do you notice how the code to read in the words is repetitive?  You could combine them into a helper method, resulting in a smaller `run_generator` and more modular code.